### PR TITLE
spoon telescope triangle wet yellow zebra

### PR DESCRIPTION
**Pull request by severin31**

### CODE REVIEW

File: data/master.tsv [hunk 1]
```diff
@@ -1743,6 +1747,8 @@ Time:wait.V	Motion:wait (for).V	To refrain from acting or moving for some durati
 Time:winter.N	Time:winter.N	Traditionally, the fourth of the seasons, marked by the applicable hemisphere of the planet being at its minimum angle of exposure to the Sun, resulting in short days, long nights and typically in low temperatures.	1471	14-74	Winter::N	3273
 Time:year.N	Time:year.N	The time it takes the Earth to complete one revolution of the Sun (between 365.24 and 365.26 days depending on the point of reference).	1226	14-73	Jahr::N	3278
 Time:yesterday.ADV	Time:yesterday.?	The day before today.	1174	14-49	gestern::ADV	3265
+Tool:spoon.N	Food:spoon.N	An implement for eating or serving, a small bowl with a long straight handle.	1378	5-37	Löffel::N	1230
```
Comment (barumau) [inline data/master.tsv:1750]: I would rather put this to the Food category. I think that there's no need for own Tool category after all.

File: data/master.tsv [hunk 2]
```diff
@@ -1743,6 +1747,8 @@ Time:wait.V	Motion:wait (for).V	To refrain from acting or moving for some durati
 Time:winter.N	Time:winter.N	Traditionally, the fourth of the seasons, marked by the applicable hemisphere of the planet being at its minimum angle of exposure to the Sun, resulting in short days, long nights and typically in low temperatures.	1471	14-74	Winter::N	3273
 Time:year.N	Time:year.N	The time it takes the Earth to complete one revolution of the Sun (between 365.24 and 365.26 days depending on the point of reference).	1226	14-73	Jahr::N	3278
 Time:yesterday.ADV	Time:yesterday.?	The day before today.	1174	14-49	gestern::ADV	3265
+Tool:spoon.N	Food:spoon.N	An implement for eating or serving, a small bowl with a long straight handle.	1378	5-37	Löffel::N	1230
+Tool:telescope.N	Tool:telescope.N	A telescope is a device used to observe distant objects by their emission, absorption, or reflection of electromagnetic radiation				1262
```
Comment (barumau) [inline data/master.tsv:1751]: This one to the Cosmos category together with _astronomy_ etc.

File: data/master.tsv [hunk 3]
```diff
@@ -96,6 +96,7 @@ Animal:Ursidae.N	Animal:bear.N	A large beast of prey of the family Ursidae, rela
 Animal:Vespinae.N	Animal:wasp.N	A flying stinging insect related to the bee, which is usually coloured yellow and black.	1517	3-823		
 Animal:Vulpes.N	Animal:fox.N	A carnivorous relatively small canine of the species Vulpes Vulpes.	1312	3-74	Fuchs::N	964
 Animal:wool.N	Clothing/Grooming:wool.N	The soft, curly hair that forms the fleece of sheep, lamas and other animals and which is used to produce clothing.	964	6-22	Wolle::N	1403
+Animal:zebra.N	Animal:zebra.N	Any of several species of genus Equus with distinctive black-and-white striped coats, native to Africa.	3675
```
Comment (barumau) [inline data/master.tsv:99]: You are right that zebra is not one species, but there is the subgenus Hippotigris for all zebras. So the id should be `Animal:Equus (Hippotigris).N`. That seems to be the standard way to write it.

**Comment by barumau** [comment]
I merge this as it is and change the mentioned ids.

File: data/master.tsv (hunk 1 above)
Comment (severin31) [reply data/master.tsv:1750]: Perfect. I agree tool shouldn't be a lexical field.

**Pull request merged**